spec/wishlist.spec.js

```diff
@@ -0,0 +1,28 @@
+import Wishlist from '../src/api/wishlist';
+
+jest.mock('../src/api/base');
+
+describe('Wishlist should add a product to the wishlist', () => {
+    const mockedUrl = '/wishlist.php?action=add&wishlistid=14&product_id=111';
+
+    test('without options', () => {
+        const wishlist = new Wishlist();
+        const callback = jest.fn();
+
+        wishlist.itemAdd('14', '111', callback);
+
+        expect(wishlist.makeRequest).toHaveBeenCalledWith(mockedUrl, 'POST', {}, false, callback);
+    });
+
+    test('with options', () => {
+        const wishlist = new Wishlist();
+        const callback = jest.fn();
```

Comment:
I would define it alongside mockedUrl

Reply:
Please address this comment

Reply:
wishlist as well can be in the top, but in that can you will need to use mocks and reset theme before each test.
look at here: https://jestjs.io/docs/setup-teardown